search/management/commands/search_maintenance.py

```diff
@@ -4,13 +4,13 @@
 
 from django.core.management.base import BaseCommand, CommandError
 from django.conf import settings
-from datetime import datetime, timedelta, date
+from datetime import datetime, timedelta, date  # noqa: F811
 import django.utils.timezone
 from server.models import *
 from search.models import *
 from inventory.models import *
 import server.utils as utils
-import datetime
+import datetime  # noqa: F811
```

Comment:
overlap between this one and line 7 above. 
This file appears to only use 'datetime' and 'timedelta' also (so maybe remove 'date' from line 7?)